sound/soc/bcm/Kconfig

```diff
@@ -166,6 +166,11 @@ config SND_BCM2708_SOC_ALLO_PIANO_DAC_PLUS
 	depends on SND_BCM2708_SOC_I2S || SND_BCM2835_SOC_I2S
 	select SND_SOC_PCM512x_I2C
 	help
+          Allo Piano DAC Plus (2.1) driver requires firmware which can
+          be obtained from:
+
+          <https://allo.com/sparky/piano-dac-2-1.html>
```

Comment:
I'm not happy about this URL - the kinds of people who won't know how to install their soundcard (which presumably comes with instructions) will never find this, and it's too much like advertising.